src/Symfony/Component/DependencyInjection/Tests/Compiler/ValidateEnvPlaceholdersPassTest.php

```diff
@@ -153,19 +153,6 @@ public function testConcatenatedEnvInConfig()
         $this->assertSame(['scalar_node' => $expected], $container->resolveEnvPlaceholders($ext->getConfig()));
     }
 
-    public function testEnvIsIncompatibleWithEnumNode()
```

Comment:
this test should be removed from DI 4.4 to fix the failures

Reply:
I created a PR #45651 to fix that.